cloudbuild_CI.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is meant to be used by the Build Triggers set in Container Builder Builds
+# of a cloud project. What it does is:
+# * Build an image using the last GitHub commit hash as the tag.
+# * Push that image to Container Registry.
+# * Runs the integration test from that very image.
+
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
```

Comment:
Optional: I prefer 's/COMMIT_SHA/TAG_NAME', and then populating TAG_NAME with COMMIT_SHA value in the script calling gcloud build.  COMMIT_SHA doesn't look to belong to here.

Reply:
Like discussed offline, these are "default substitutions" in Container Builder:
https://cloud.google.com/container-builder/docs/configuring-builds/substitute-variable-values
So COMMIT_SHA is the commit hash value at which the image was created and tested.